experimenter/experimenter/nimbus_ui/templates/nimbus_experiments/timeline.html

```diff
@@ -1,6 +1,9 @@
 <ul class="list-group list-group-horizontal justify-content-between mb-3">
   {% for status in experiment.timeline %}
-    <li class="list-group-item flex-fill text-center d-flex flex-column justify-content-center {% if status.is_active %}bg-primary text-white{% endif %} {% if status.step > experiment.experiment_active_status %}bg-secondary text-muted{% endif %}">
+    <li class="list-group-item flex-fill text-center d-flex flex-column justify-content-center {% if status.is_active %}bg-primary text-white{% endif %}"
+        style="{% if status.step > experiment.experiment_active_status %}background-color: var(--bs-gray-500);
+               color: var(--bs-gray-700);
```

Comment:
Oh in general it's better to avoid explicitly setting the `style` property when there's bootstrap classes you can use instead, becuase they'll have rules that interact with other classes nicely. 

You can find the [bootstrap background classes here](https://getbootstrap.com/docs/5.0/utilities/background/#background-color) maybe `.bg-secondary` is a good fit?